### DISCUSSION TITLE
Dutch (nld) OCR

### DISCUSSION THREAD

**Discussion by Bert-Jan53** [papermerge/papermerge-core#320] [Ideas]
Hi,
I realise a new release with 7 OCR languages is on the horizon, so I'm too late for now, but would it be possible to add Dutch to an upcoming release?
I'm new to Github, so if I need to ask this question somwhere else, please let me know.

Kind regards,
Bert-Jan

**Comment by ciur** [comment]
Ok, I will add Dutch as well
